tests/src/main/java/org/hibernate/beanvalidation/tck/tests/constraints/constraintdefinition/Movie.java

```diff
@@ -0,0 +1,40 @@
+/*
+* JBoss, Home of Professional Open Source
+* Copyright 2013, Red Hat, Inc. and/or its affiliates, and individual contributors
+* by the @authors tag. See the copyright.txt in the distribution for a
+* full listing of individual contributors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+* http://www.apache.org/licenses/LICENSE-2.0
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+package org.hibernate.beanvalidation.tck.tests.constraints.constraintdefinition;
+
+import javax.validation.constraints.Size;
+
+/**
+ * @author Gunnar Morling
```

Comment:
Yeah, I copied it from another test and wasn't sure about changing the authorship just by changing the constraints annotation. I changed it to me as it looks like you don't want this burden on your shoulders :).